content/docs/gke/gcp-e2e.md

```diff
@@ -133,22 +133,16 @@ Run the following Cloud SDK command to install the
 gcloud components install kubectl
 ```
 
-### Install ksonnet
+### Install kustomize
 
-Kubeflow makes use of [ksonnet] to help manage deployments. ksonnet acts as a
-layer on top of `kubectl`. While Kubernetes is typically managed with static
-YAML files, ksonnet adds a further abstraction that is closer to the objects
-in object-oriented programming.
+Kubeflow makes use of [kustomize](https://github.com/kubernetes-sigs/kustomize) to help manage deployments. 
 
-With ksonnet, you manage your resources as *prototypes* with empty parameters.
-Then you instantiate the prototypes into *components* by defining values for the
-parameters. This system makes it easier to deploy slightly different resources
-to different clusters at the same time. In this way you can maintain different
-environments for staging and production, for example.
+kustomize lets you customize raw, template-free YAML files for multiple purposes, leaving the original YAML untouched and usable as is.
 
-Make sure you have ksonnet version {{% ksonnet-min-version %}} or later.
-See the [ksonnet component guide](/docs/components/ksonnet) for help with
-installing ksonnet.
+kustomize targets kubernetes; it understands and can patch kubernetes style API objects. It's like `make`, in that what it does is declared in a file, and it's like `sed`, in that it emits editted text.
```

Comment:
Let's remove this paragraph. It's copied directly from the [kustomize README](https://github.com/kubernetes-sigs/kustomize/blob/master/README.md), and it doesn't add anything useful to this Kubeflow doc.